engine/src/field.h

```diff
@@ -241,6 +241,7 @@ class MCField : public MCControl, public MCMixinObjectHandle<MCField>
     MCTextDirection text_direction;
     MCInterfaceFieldCursorMovement cursor_movement;
     intenum_t keyboard_type;
+    intenum_t return_key_type;
```

Comment:
This should be the explicit MCInterfaceReturnKeyType enum - also can be a bitfield.